test_esss_jenkins.py

```diff
@@ -230,14 +231,43 @@ def test_bhist(jenkins_plugin, testbot, mocker, LineMatcher):
 
 
 def test_fetch_job_status(jenkins_plugin, mocker):
```

Comment:
Please make this test parametrized. 😉

Reply:
Done, just couldn't figure out how to parametrize the status "NOT_STARTED", so created a new test.